DBSchema.sql

```diff
@@ -5,6 +5,7 @@ CREATE TABLE IF NOT EXISTS conversations (
     ID INTEGER NOT NULL AUTO_INCREMENT,
     Name VARCHAR(255),
     Description VARCHAR(255),
+	AvatarURL VARCHAR(255),
```

Comment:
Change it from a tab to 4 spaces, so that it lines up with the other ones.
```suggestion
    AvatarURL VARCHAR(255),
```